src/utils/baseUrl.js

```diff
@@ -1 +1,3 @@
-export const baseUrl = "https://cranky-kepler-2a0ccc.netlify.app/api";
+// export const baseUrl = "http://localhost:3000/api"; // development dummy API
+
+export const baseUrl = "https://cranky-kepler-2a0ccc.netlify.app/api"; // production dummy API
```

Comment:
you can learn the environment variable for the next time, the point is how to make the value changing based on what the environment

source:
https://nextjs.org/docs/basic-features/environment-variables

Reply:
okay kak, i forgot it